packages/api/src/routes/taskRoute.js

```diff
@@ -38,6 +38,13 @@ router.patch(
   taskController.assignAllTasksOnDate(),
 );
 
+router.patch(
```

Comment:
Since you are adding an API route, you should add jest test cases to automatically test it. I can help you set that up if it's unfamiliar to you.